pkg/server/admin.go

```diff
@@ -554,7 +554,21 @@ func (s *adminServer) TableStats(
 
 	// Construct TableStatsResponse by sending an RPC to every node involved.
 	tableStatResponse := serverpb.TableStatsResponse{
-		NodeCount:  int64(len(nodeIDs)),
+		NodeCount: int64(len(nodeIDs)),
+		// TODO(mrtracy): The "RangeCount" returned by TableStats is more
```

Comment:
can you please make sure this "TODO" actually includes a task description? currently it seems only to describe the status quo which makes it impossible for anyone but you to act on.